src/main/kotlin/com/goodchoice/domain/review/model/VoteType.kt

```diff
@@ -0,0 +1,19 @@
+package com.goodchoice.domain.review.model
+
+enum class VoteType {
+    UP, DOWN
+}
+
+fun VoteType.asJooqVoteType(): com.goodchoice.domain.common.jooq.enums.VoteType {
+    return when (this) {
+        VoteType.DOWN -> com.goodchoice.domain.common.jooq.enums.VoteType.DOWN
+        VoteType.UP -> com.goodchoice.domain.common.jooq.enums.VoteType.UP
+    }
+}
+
+fun com.goodchoice.domain.common.jooq.enums.VoteType.asVoteType(): VoteType {
```

Comment:
We can use `import com.goodchoice.domain.common.jooq.enums.VoteType as JooqVoteType` in this file to avoid full name specification

Reply:
done